lib/teracy-dev-v05-compat/trigger/rsync_trigger.rb

```diff
@@ -0,0 +1,44 @@
+require 'teracy-dev/config/configurator'
```

Comment:
let's rename and move the `rsync_trigger.rb` to `lib/config/gatling-rsync-recovery.rb`
```ruby
GatlingRsyncRecovery < Configurator
```